snooty/test_util.py

```diff
@@ -37,6 +37,7 @@ def test_get_files() -> None:
         Path("test_data/bad_project/snooty.toml"),
         Path("test_data/empty_project/snooty.toml"),
         Path("test_data/test_project/snooty.toml"),
+        Path("test_data/test_project_ls/snooty.toml"),
```

Comment:
I'm not sure `test_project_ls` is sufficiently descriptive. I might go for something along the lines of `test_project_embedding_includes` or such